tests/test_sft_trainer.py

```diff
@@ -980,6 +980,48 @@ def test_run_chat_style_ft(dataset_path):
         assert 'Provide two rhyming words for the word "love"' in output_inference
 
 
+@pytest.mark.parametrize(
+    "data_args",
+    [
+        (
+            # sample hugging face dataset id
+            configs.DataArguments(
+                training_data_path="lhoestq/demo1",
+                data_formatter_template="### Text:{{review}} \n\n### Stars: {{star}}",
+                response_template="\n### Stars:",
+            )
+        )
+    ],
+)
+def test_run_chat_style_add_special_tokens_ft(data_args):
+    """Check if we can perform an e2e run with chat template and multi turn chat training."""
+    with tempfile.TemporaryDirectory() as tempdir:
+
+        data_args.add_special_tokens = ["<|assistant|>", "<|user|>"]
```

Comment:
Can you move this to data argument initialisation.
Also since this is a single test you can remove the parametrize.

Reply:
Sure